private/pkg/app/app.go

```diff
@@ -136,13 +136,31 @@ func NewArgContainerForOS() ArgContainer {
 	return newArgContainer(os.Args)
 }
 
+// FeatureFlag represents an application feature flag.
+type FeatureFlag struct {
```

Comment:
Why is this and `FeatureContainer` needed? If this is something temporary, it feels like you just read from an `EnvContainer`. I don't think this addition to the `app` package is desired or necessary (in general, CLI feature flagging isn't something we want to encourage)

Reply:
Removed.